include/aos/common/config.hpp

```diff
@@ -155,4 +155,88 @@
 #define AOS_CONFIG_MONITORING_POLL_PERIOD_SEC 10
 #endif
 
+/**
+ * Certificate public key len.
+ */
+#ifndef AOS_CONFIG_CRYPTO_CERT_PUB_KEY_LEN
+#define AOS_CONFIG_CRYPTO_CERT_PUB_KEY_LEN 2048
+#endif
+
+/**
+ * Certificate private key len.
+ */
+#ifndef AOS_CONFIG_CRYPTO_CERT_PRIV_KEY_LEN
+#define AOS_CONFIG_CRYPTO_CERT_PRIV_KEY_LEN 2048
+#endif
+
+/**
+ * Certificate serial number len.
+ */
+#ifndef AOS_CONFIG_CRYPTO_CERT_SERIAL_LEN
+#define AOS_CONFIG_CRYPTO_CERT_SERIAL_LEN 40
+#endif
+
+/**
+ * Certificate issuer len.
+ */
+#ifndef AOS_CONFIG_CRYPTO_CERT_ISSUER_ID_LEN
```

Comment:
It is not ID. Issuer is rather a struct with additional fields inside. Rename to `AOS_CONFIG_CRYPTO_CERT_ISSUER_SIZE`

40 definitely is not enough: let's put 256 for now.

Use `LEN` for string values and `SIZE` for other.